packages/core/src/external/carequality/ihe-gateway-v2/outbound/__tests__/process-xcpd.test.ts

```diff
@@ -126,3 +129,97 @@ describe("processXCPDResponse", () => {
     expect(response.patientMatch).toBeNull();
   });
 });
+
+describe("convertIheAddressToAddress", () => {
+  it("should correct invalid streetAddressLine", () => {
+    const iheAddresses = {
+      streetAddressLine: [2831],
+      city: "San Francisco",
+      state: "CA",
+      postalCode: "94101",
+      country: "USA",
+    };
+    const result = convertIheAddressToAddress(iheAddresses);
+    expect(result).toEqual({
+      line: ["2831"],
+      city: "San Francisco",
+      country: "USA",
+      postalCode: "94101",
+      state: "CA",
+    });
+    if (result) {
+      const isValid = isValidAddress(result);
+      expect(isValid).toBeTruthy();
```

Comment:
The best practices about unit tests indicate they should validate a single thing.

For example, right now we're only testing addresses that should be valid, but not ones that shouldn't -  and this is important too, because we don't want to break b/c of those.